webpack.config.js

```diff
@@ -177,7 +177,11 @@ const config = {
 					if ( config.mode === 'production' ) {
 						return postcss( [
 							require( 'cssnano' )( {
-								preset: 'default',
+								preset: [ 'default', {
+									discardComments: {
+							                    removeAll: true,
+							                },
+								}]
```

Comment:
Looks like there's some failing eslint rules here. You can run `npm run lint` locally to test.